graphql-mode.el

```diff
@@ -86,6 +89,8 @@ response from the server."
     (forward-line 1)))
 
 (defun graphql-current-query ()
+  "find out the current query/mutation/subscription"
+  (interactive)
```

Comment:
Does it need to be interactive though?

Reply:
No, `graphql-current-query` doesn't need to be a command. (I was testing out command manually). will revert it back to function.